### ISSUE TITLE
 ram_max_used fix ineffective on updated devices due to persistent storage 

### ISSUE THREAD

**Issue by ptrm** [balenalabs/rosetta-at-home#29]
As the /usr/app/boinc folder became a volume, the changes fixing #25 do not come to life on existing instances:
![Screenshot from 2020-04-24 18-10-34](https://user-images.githubusercontent.com/366258/80233730-3fddd200-8657-11ea-81c9-c7e2ab9fd9d7.png)

Only one of the global override files was edited, but this seems to not matter.

**Comment by ptrm** [comment]
...which can probably be fixed for balena-managed devices from foldforcovid.io using rosetta project preferences.

**Comment by chrisys** [comment]
@ptrm ah yes, we should add a sed action to update existing devices - I'll do another PR for that. Thanks for your continued work testing this project! :)

**Comment by ptrm** [comment]
Thanks! I myself started scribbling a bash script with checks and substitutions only if max ram is greater than 95% and same for cpu. Also with a env variable with flags to skip the checks. Should come around the weekend

**Comment by chrisys** [comment]
@ptrm by all means PR your work, I'd love to include it!

**Comment by chrisys** [comment]
Just a note, if you do decide to PR, take a look at the contribution guidelines for balenaSound, the same will apply here with regards to the CI: https://github.com/balenalabs/balena-sound/blob/master/CONTRIBUTING.md

**Comment by ptrm** [comment]
Added a pull request, though I have to admit that after #27 my 4GB Pi 4 still loses ui from time to time but is still steadily working. And the 2GB Pi 4 reboots with more than one task - tried forcing 3 and 2 cores as well.

And sorry for the merge commit - forgot to sync with the root repo before creating the PR.

**Comment by chrisys** [comment]
@ptrm great work, I was just testing your PR and seeing the same reboot issue on my 2GB Pi 4 unfortunately, so perhaps we should put the limit back up to 2500000 as it was before.
